adserver/models.py

```diff
@@ -282,6 +282,17 @@ class Publisher(TimeStampedModel, IndestructibleModel):
         blank=True,
     )
 
+    # If this is blank, all domains are allowed
+    allowed_domains = models.CharField(
+        _("Allowed domains"),
+        max_length=1024,
+        help_text=_(
+            "A space separated list of domains where the publisher's ads can appear"
```

Comment:
Do we allow wildcards or similar here? Or is it a `if in` check or similar? Could be useful to be more specific so we know what to include, or is it an exact match?

Reply:
I think we may need those at some point. Before we implement it though, I'd like to know how common that is.